src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfacePrompt.cs

```diff
@@ -306,7 +306,8 @@ public override
                         null,   // caption already written
                         null,   // message already written
                         null,
-                        string.Empty);
+                        string.Empty,
+                        false);
```

Comment:
Please use named arguments for all constants and nulls like
```suggestion
                        confirmPassword: false);
```